knora/dsplib/utils/excel_to_json_lists.py

```diff
@@ -253,7 +253,8 @@ def validate_lists_section_with_schema(
     """
     if bool(path_to_json_project_file) == bool(lists_section):
         raise BaseError("Validation of the 'lists' section works only if exactly one of the two arguments is given.")
-    with open("knora/dsplib/schemas/lists-only.json") as schema:
+    current_dir = os.path.dirname(os.path.realpath(__file__))
+    with open(os.path.join(current_dir, "../schemas/lists-only.json")) as schema:
```

Comment:
The old version only worked because during development, I always invoke commands from the directory `dsp-tools` (=working directory). As soon as the working directory is different, it doesn't work any more. I wonder why it worked for our customers until now...

Reply:
did you install it with `-e` flag maybe? in any case, if that fixes it, all the better

Reply:
Yes, I thought that would be the only sensible way to install it for development. The point is that I often want to test a dsp-tools command from terminal, and I want that it takes the one I am editing/developing currently.

Reply:
sure, makes sense, but of course this may change the behaviour of relative paths compared to how users experience it